2-multitenant/README.md

```diff
@@ -1,2 +1,54 @@
 # Multitenant Infrastructure phase
-This phase deploys multitenant infrastruture using the multitenant infrastructure pipeline.
+
+## Purpose
+
+This phase deploys the per-environment multitenant resources deployed via the multitenant infrastructure pipeline.
+
+The following resources are created:
+- GCP Projects
+- GKE cluster(s)
+- Cloud SQL PostgreSQL
+- Cloud Endpoint
+- Cloud Armor
+- IP addresses
+- Service Accounts
```

Comment:
I wouldn't mention service accounts unless they really must be explained on why they are created?